### DISCUSSION TITLE
Airyx build hammering on my homelab Proxmox server

### DISCUSSION THREAD

**Discussion by markshank** [ravynsoft/ravynos#134] [Show and tell]
![make prep airyx 2021-09-26 113242](https://user-images.githubusercontent.com/25345979/134815295-baa0138b-c035-4b0e-8ef5-920d9be075f2.png)

**Comment by grahamperrin** [comment]
Off-topic, I see <http://nmon.sourceforge.net/> however nmon is not in FreshPorts or at <https://wiki.freebsd.org/WantedPorts>. 

What's the 'closest' comparable package for FreeBSD?

**Reply by markshank** [reply]
I don't know, someone should port it to FreeBSD.  The nmon session displayed above is running on the Proxmox host hypervisor (Debian Bullseye) via SSH.  That's why it shows 16 cpus while my Airyx VM only has 12.

Once upon a time, I wanted a tool to monitor disk activity on my [Ceph nodes](https://pve.proxmox.com/wiki/Deploy_Hyper-Converged_Ceph_Cluster).  I tried nmon and it was just the ticket.  It also helps me identify memory hungry processes when ram gets tight.

**Reply by davidchisnall** [reply]
I've not seen `nmon` before, but this particular use of it at least is covered by [`systat`](https://www.freebsd.org/cgi/man.cgi?query=systat).
